src/resource.js

```diff
@@ -1,5 +1,5 @@
 import Request from './request';
-import {Container} from './';
+// import descriptor from './descriptor';
```

Comment:
No comment here.

Reply:
ok, i had already removed the comments on the last commit.